From 225f3f649a366dbffcf74692c6a29ff898d23be8 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Sat, 6 May 2017 17:10:07 +0800
Subject: [PATCH 45/80] dt-bindings: add bindings for the A64 display pipeline

Add compatible strings for the following devices in A64 display
pipeline:
- Channel1-less TCON0 and Channel0-less TCON1
- A64 Display Engine 2.0

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 Documentation/devicetree/bindings/display/sunxi/sun4i-drm.txt | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/sunxi/sun4i-drm.txt b/Documentation/devicetree/bindings/display/sunxi/sun4i-drm.txt
index 44b61658b8af..84739924bc77 100644
--- a/Documentation/devicetree/bindings/display/sunxi/sun4i-drm.txt
+++ b/Documentation/devicetree/bindings/display/sunxi/sun4i-drm.txt
@@ -89,6 +89,8 @@ Required properties:
    * allwinner,sun8i-a33-tcon
    * allwinner,sun8i-h3-tcon
    * allwinner,sun8i-v3s-tcon
+   * allwinner,sun50i-a64-tcon0
+   * allwinner,sun50i-a64-tcon1
  - reg: base address and size of memory-mapped region
  - interrupts: interrupt associated to this IP
  - clocks: phandles to the clocks feeding the TCON. Three are needed:
@@ -117,6 +119,7 @@ For the following compatibles:
    * allwinner,sun6i-a31s-tcon
    * allwinner,sun8i-a33-tcon
    * allwinner,sun8i-v3s-tcon
+   * allwinner,snu50i-a64-tcon0
 there is one more clock and one more property required:
  - clocks:
    - 'tcon-ch0': The clock driving the TCON channel 0
@@ -127,6 +130,7 @@ For the following compatibles:
    * allwinner,sun6i-a31-tcon
    * allwinner,sun6i-a31s-tcon
    * allwinner,sun8i-h3-tcon
+   * allwinner,sun50i-a64-tcon1
 there is one more clock required:
    - 'tcon-ch1': The clock driving the TCON channel 1
 
@@ -224,6 +228,8 @@ Required properties:
     * allwinner,sun8i-v3s-de2-mixer
     * allwinner,sun8i-h3-de2-mixer0
     * allwinner,sun8i-h3-de2-mixer1
+    * allwinner,sun50i-a64-de2-mixer0
+    * allwinner,sun50i-a64-de2-mixer1
   - reg: base address and size of the memory-mapped region.
   - clocks: phandles to the clocks feeding the mixer
     * bus: the mixer interface clock
@@ -251,6 +257,7 @@ Required properties:
     * allwinner,sun8i-a33-display-engine
     * allwinner,sun8i-h3-display-engine
     * allwinner,sun8i-v3s-display-engine
+    * allwinner,sun50i-a64-display-engine
 
   - allwinner,pipelines: list of phandle to the display engine
     frontends (DE 1.0) or mixers (DE 2.0) available.
-- 
2.13.6

